bin/all_sky_search/pycbc_sngls_findtrigs

```diff
@@ -114,14 +116,29 @@ template_ids = cuts.apply_template_cuts(
     template_ids=template_ids)
 
 logging.info("%d out of %d templates kept after applying template cuts",
-             len(template_ids), len(trigs.bank))
+             len(template_ids), original_bank_len)
 
 logging.info('Clustering events over %s s window within each template',
              args.cluster_window)
+
+# Stat class instance to calculate the ranking statistic
```

Comment:
Move the `extra_kwargs` definition out of the loop - not much saving computationally but best practice